get_notebook.py

```diff
@@ -0,0 +1,8 @@
+import os
+import galaxy
+import shutil
+
+hid = os.environ.get('DATASET_HID', None)
+if hid not in ('None', None):
+    galaxy.get(int(hid))
+    shutil.copy('/import/%s' % hid, '/import/ipython_galaxy_notebook.ipynb')
```

Comment:
We wanted to have intentionally two copies. One the original and one the current working copy.